### ISSUE TITLE
Larger than usual nav menu icon on Toolbar

### ISSUE THREAD

**Issue by plusCubed** [afollestad/cabinet#117]
Also, after opening up the nav drawer and closing it halfway to see the nav menu icon on the toolbar, it changed into a back icon. It instantly becomes the menu icon again after completely closing the nav drawer.

**Comment by afollestad** [comment]
Oh yeah, very weird.

It's not larger than usual, though. I don't specify an icon, ActionBarDrawerToggle uses a default nav drawer icon that Google provides in AppCompat.

**Comment by afollestad** [comment]
However, I disabled the display of that arrow when the nav drawer is open.

**Comment by plusCubed** [comment]
I'll post a comparison image between Gmail and Cabinet/Impression later (you can check it out yourself), but the ActionBarDrawerToggle in AppCompat is definitely the wrong size. If the animation isn't going to be used in Cabinet, I suggest reverting back to setting a static icon. (Interesting thing though: the Play Store has the toggle animation, so I presumed they used the ActionBarDrawerToggle, but their menu icon size is correct compared to Gmail and the I/O 2014 app. Inbox has the toggle animation too, but as expected is oversized.)

**Comment by plusCubed** [comment]
Also check out this in the material-design-icons repo: https://github.com/google/material-design-icons/blob/master/navigation/drawable-xxhdpi/ic_menu_white_24dp.png

**Comment by afollestad** [comment]
Gmail's isn't necessarily correct, AppCompat is by Google engineers so I doubt they'd use the wrong icon. Inbox and Impression/Cabinet are using the same icon, too. Google Play, too, I believe.

![collage](https://cloud.githubusercontent.com/assets/1820165/5989974/13326e6a-a961-11e4-98ac-95aaf5976a68.jpg)

It looks fine IMO.

**Comment by plusCubed** [comment]
I checked out some more Google apps, and I came to this conclusion: Google is either inconsistent or has a Civil War of the Nav Icons going on. 
The two sides:
- Gmail, Google Play, Google Calendar, Google Docs, Sheets, Slides, Play Games, Keep, Hangouts, Google Search, Google Maps, Wallet
- Play Music, Inbox, Youtube, Play Books, Play Newsstand, Google+ Photos, basically any that uses the ActionBarDrawerToggle
Then there's Google Drive is doing its own thing with the icon.

Anyway, I suppose it's fine for now, since even Google can't decide.
